modules/exploits/multi/http/kong_gateway_admin_api_rce.rb

```diff
@@ -0,0 +1,138 @@
+# frozen_string_literal: true
+
+##
```

Comment:
give the module a `rubocop` run, 5 things to clean up.

Reply:
`rubocop` complains about 5 things. Do all of these need to be satisfied?

- Metrics/ClassLength: Class has too many lines
- Style/Documentation: Missing top-level class documentation comment
- Metrics/MethodLength: Method has too many lines
- Metrics/AbcSize: Assignment Branch Condition size for check is too high
- Metrics/MethodLength: Method has too many lines